components/eventing-controller/testing/subscriber.go

```diff
@@ -73,6 +174,7 @@ func getServeMux() *http.ServeMux {
 		retries.Inc()
 		w.WriteHeader(http.StatusInternalServerError)
 	})
+	// this endpoint returns the number of attempted retries.
```

Comment:
(nit)
for consistency 
```suggestion
	// this Endpoint returns the number of attempted retries.
```